docs/source-2.0/spec/model.rst

```diff
@@ -789,7 +789,7 @@ target from traits and how their values are defined in
       - number | string
       - If a number is provided, it represents Unix epoch seconds with optional
         millisecond precision. If a string is provided, it MUST be a valid
-        :rfc:`3339` string with no UTC offset and optional fractional
+        :rfc:`3339` string with optional fractional precision but no UTC offset
```

Comment:
Looks like the next line still has the word "precision", but should not

Reply:
Makes sense, updated. Thanks!